Directory.Packages.props

```diff
@@ -2,27 +2,27 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <AvaloniaVersion>11.0.0</AvaloniaVersion>
+    <AvaloniaVersion>11.0.1</AvaloniaVersion>
```

Comment:
We prefer to keep a lowest possible version for dependencies. Actually, up to date version should be only in demo project.

Reply:
Sure